packages/live-share-media/src/test/LiveMediaSession_ManualActionHandler.spec.ts

```diff
@@ -246,9 +245,13 @@ describeNoCompat(
                 }
             });
 
+            await object1.initialize();
```

Comment:
should we change the name of the test to something like `position updates are sent on joined event`?

Reply:
I think we already have a UT that does that

Reply:
Yeah we have one called "should send 'positionUpdate' event when someone joins", whereas this one is supposed to be testing whether it is sent periodically (that was my impression)